### DISCUSSION TITLE
Multiple Game Install locations

### DISCUSSION THREAD

**Discussion by tacogerbil** [PhilipK/BoilR#206] [Ideas]
Once you start running out of storage you are unable to start using an external usb drive as an additional game location.  I know there is already an option to change install location for GOG games, but there is no info as to whether this will break the prior game installations.

**Comment by PhilipK** [comment]
The paths that BoilR uses are usually not pointing to where the games are
installed, but instead to the configuration files of the launcher..
For epic it is looking for manifest files.
For itch its a butler.db file.
For Amazon a sqlite database.

And so on and so on.
So you should be able to install games in other locations as long as the
launcher (not boilr) supports it.

**Reply by tacogerbil** [reply]
You can install.. I had Heroic install to my SSD USB drive.. Plays fine from Heroic.  Boilr won't pick up those games though..

**Comment by PhilipK** [comment]
Hmm I have a feeling this is a flatpak permission thing again. Have you
tried the native version of Boilr? Can it find them?

**Reply by tacogerbil** [reply]
Thank you for pointing me in the right direction :)
I gave the app permission to access the /run/media folder and **bammo**! it read them just fine :)
